src/pages/SearchList/SearchList.js

```diff
@@ -1,8 +1,293 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
+import * as s from "./Styled.SearchList";
+import CalendarModal from "./Calendar";
+import LocationModal from "./LocationModal";
+import HeadcountModal from "./HeadcountModal";
+import { faLocationDot } from "@fortawesome/free-solid-svg-icons";
+import { faCalendar } from "@fortawesome/free-solid-svg-icons";
+import { faUserLarge } from "@fortawesome/free-solid-svg-icons";
 
 const Searchlist = () => {
-  return <div>SearchList</div>;
+  const [calendar, setCalander] = useState(false);
+  const [location, setLocation] = useState(false);
+  const [headcount, setHeadcount] = useState(false);
```

Comment:
boolean 데이터를 가지는 변수는 앞에 is를 붙여주세요!
```suggestion
  const [isCalendar, setIsCalander] = useState(false);
  const [isLocation, setIsLocation] = useState(false);
  const [isHeadcount, setIsHeadcount] = useState(false);
```

Reply:
수정 완료